### PR TITLE
validate payment in circuitos page

### PR DESCRIPTION
**Pull request by JEdilbertoTC**

### CODE REVIEW

File: backend/src/main/java/org/uresti/pozarreal/controllers/PaymentsController.java [hunk 1]
```diff
@@ -63,4 +63,15 @@ public void deletePayment(@PathVariable String paymentId, Principal principal) {
 
         paymentsService.delete(paymentId);
     }
+
+    @PatchMapping("/{paymentId}")
+    @ResponseStatus(HttpStatus.OK)
```
Comment (iuresti) [inline backend/src/main/java/org/uresti/pozarreal/controllers/PaymentsController.java:68]: The default response is HttpStatus.OK .. no need to add this line

File: frontend/src/app/components/circuitos/circuitos.component.ts [hunk 2]
```diff
@@ -51,8 +55,8 @@ export class CircuitosComponent implements OnInit {
         this.selectStreet();
       } else {
         this.activatedRoute.params.subscribe(params => {
-          if (params['streetId']) {
-            this.selectedStreetId = params['streetId'];
+          if (params.streetId) {
```
Comment (iuresti) [inline frontend/src/app/components/circuitos/circuitos.component.ts:58]: what's the reason of this change?

**Review by iuresti — APPROVED** [review]

File: frontend/src/app/components/circuitos/circuitos.component.ts (hunk 2 above)
Comment (JEdilbertoTC) [reply frontend/src/app/components/circuitos/circuitos.component.ts:58]: TSLINT is pressing

File: frontend/src/app/services/payment.service.ts [hunk 3]
```diff
@@ -81,4 +81,7 @@ export class PaymentService {
     return this.http.get<Payment>(`${environment.baseUrl}/payments/${id}`);
   }
 
+  updateValidated(id: string): Observable<Payment> {
```
Comment (iuresti) [inline frontend/src/app/services/payment.service.ts:84]: Looking to this name anyone should see the implementation to understand what's the meaning of "updateValidated" ... please change the name to something like "validatePayment" or "setValidatedToTrue" (I prefer the first option).

Remember your code must be auto-documented

**Review by iuresti — CHANGES_REQUESTED** [review]

File: frontend/src/app/services/payment.service.ts (hunk 3 above)
Comment (JEdilbertoTC) [reply frontend/src/app/services/payment.service.ts]: Ok, i did

**Review by iuresti — APPROVED** [review]
